app/views/account/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Your account</h1>
+
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          First name
```

Comment:
Should we use translations for these?

Reply:
Resolved in [9f8f3b0](https://github.com/DFE-Digital/itt-mentor-services/pull/12/commits/9f8f3b07860c747a02a99e58c3b84c360cae9354)